share/composer/vhd.ks

```diff
@@ -34,6 +34,27 @@ rm /var/lib/systemd/random-seed
 # Clear /etc/machine-id
 rm /etc/machine-id
 touch /etc/machine-id
+
+# Set up network.
+cat > /etc/sysconfig/network-scripts/ifcfg-eth0 << EOF
+DEVICE=eth0
+ONBOOT=yes
+BOOTPROTO=dhcp
+TYPE=Ethernet
+USERCTL=yes
+PEERDNS=yes
+IPV6INIT=no
+EOF
+
+# Add Hyper-V modules into initramfs
+cat > /etc/dracut.conf.d/10-hyperv.conf << EOF
+add_drivers+=" hv_vmbus hv_netvsc hv_storvsc "
+EOF
+
+dracut -f -v --persistent-policy by-uuid
```

Comment:
Do you know why we're doing this? Can we add a comment?

Reply:
Fixed.

Reply:
We really shouldn't run dracut from post. Ideally these extra drivers would be added by a package, or maybe (ugh) a kickstart dracut command. I guess it's fine for the moment, but we should find a better way to do this.